### ISSUE TITLE
Use mocha instead of jest in `markdown-docx`

### ISSUE THREAD

**Issue by algomaster99** [accordproject/markdown-transform#403]
# Feature Request 🛍️

Use `mocha` to execute tests in `markdown-docx` instead of `jest `.

## Use Case

Mocha gives a clearer output of what tests passed. Eg.
**Mocha**
![Screenshot from 2021-06-11 21-55-17](https://user-images.githubusercontent.com/35191225/121718253-bcec3500-caff-11eb-8432-f60a5b2f8ea9.png)
**Jest**
![Screenshot from 2021-06-11 21-56-07](https://user-images.githubusercontent.com/35191225/121718368-dc835d80-caff-11eb-8e5a-64a656157bb5.png)

## Possible Solution

- Put all the tests in `test` directory.
- Change `npm test` script to `mocha` instead of `jest`.

**Comment by K-Kumar-01** [comment]
@algomaster99 
I was able to convert all but one test file to run with mocha. It is primarily due to the fact that it uses a snapshot test.
![Screenshot from 2021-06-12 23-25-29](https://user-images.githubusercontent.com/59891164/121785114-7f59db80-cbd5-11eb-9b46-1e8063f3dd6a.png)
![Screenshot from 2021-06-12 23-23-40](https://user-images.githubusercontent.com/59891164/121785061-44f03e80-cbd5-11eb-8f96-13118ed931a7.png)
Mocha does not support snapshot testing itself so I am using [this package](https://www.npmjs.com/package/mocha-chai-snapshot) 

The snapshots are created already. I have not touched it. 
I tried moving snapshots outside of src(in the root) and running but got the same error.
